src/backbone.syphon.typeregistry.js

```diff
@@ -17,7 +17,11 @@ _.extend(TypeRegistry.prototype, {
   // found for the specified type, the default is
   // returned.
   get: function(type){
-    return this.registeredTypes[type] || this.registeredTypes['default'];
+    if (this.registeredTypes.hasOwnProperty(type)){
```

Comment:
heh if registeredTypes is a host object it won't work :) but i guess registeredTypes will never be a host object